tests/context_tests.rs

```diff
@@ -1257,6 +1257,69 @@ mod test_policy_pcr {
     }
 }
 
+mod test_change_auth {
+    use super::*;
+
+    #[test]
+    fn test_object_change_auth() {
+        let mut context = create_ctx_with_session();
+
+        let prim_key_handle = context
+            .create_primary_key(
+                Hierarchy::Owner,
+                &decryption_key_pub(),
+                None,
+                None,
+                None,
+                PcrSelectionListBuilder::new().build(),
```

Comment:
Seeing this provided in each test for each `create..`, I'm in two minds about potentially changing this to an `Option` too, so we can just pass `None`

Reply:
Yep, makes sense. I can create a PR for that.

Reply:
No need to do that now, I'll raise an issue

Reply:
:+1: